lib/ViewOrganizationCard/ViewOrganizationCard.js

```diff
@@ -139,16 +141,16 @@ export default class ViewOrganizationCard extends React.Component {
   renderOrganizationName = (orgName) => {
     const { id } = this.props;
     return (
-      <span>
-        <AppIcon
-          app="organizations"
-          size="small"
-        >
-          <strong id={id} data-test-org-name>
-            {orgName}
+      <Col>
```

Comment:
Cols need to be children of Rows, not vice versa.

Reply:
Whoops, I've switched them now